tests/api/parser_v200_verify_test.go

```diff
@@ -80,27 +56,6 @@ func Test_CompositeCommand(t *testing.T) {
 func Test_CompositeCommandEdit(t *testing.T) {
 	testContent := TestContent{}
 	testContent.CommandTypes = []schema.CommandType{schema.CompositeCommandType}
-	testContent.CreateWithParser = false
-	testContent.EditContent = true
-	testContent.FileName = utils.GetDevFileName()
-	runTest(testContent, t)
-	runMultiThreadTest(testContent, t)
-}
-
-func Test_CompositeCommandParserCreate(t *testing.T) {
-	testContent := TestContent{}
-	testContent.CommandTypes = []schema.CommandType{schema.CompositeCommandType}
-	testContent.CreateWithParser = true
-	testContent.EditContent = false
-	testContent.FileName = utils.GetDevFileName()
-	runTest(testContent, t)
-	runMultiThreadTest(testContent, t)
-}
-
-func Test_CompositeCommandEditParserCreate(t *testing.T) {
-	testContent := TestContent{}
-	testContent.CommandTypes = []schema.CommandType{schema.CompositeCommandType}
-	testContent.CreateWithParser = true
 	testContent.EditContent = true
 	testContent.FileName = utils.GetDevFileName()
```

Comment:
minor suggestion: 
I saw all the tests created are setting those values. can create a func `newTestContent()`
```
newTestContent(commandType, editContent) {
	testContent := TestContent{}
	testContent.CommandTypes = commandType
	testContent.EditContent = editContent
	testContent.FileName = utils.GetDevFileName()

        return testContent
}

```

Reply:
The kicker here is that utils.GetDevFileName() creates a filename based on the calling function name so it needs to be in the test function. I could update it to walk the stack until it finds the Test_* function but would rather not in this pull request.  Also I have though of similar in the past but just thought it would not save much.